### PR TITLE
Move folder to reflect package name properly

### PR DESCRIPTION
**Pull request by erasche**
skip ci

### CODE REVIEW

**Comment by nsoranzo** [comment]
I think it should be the other way round, i.e. changing the name in `.shed.yml` to `rgrnastar_240d`.

Or even better, just drop the version from the tool name and from the directory name!

**Comment by erasche** [comment]
The .shed.yml file says "rna STAR 2.0.3e" hence moving it to that name. I'd definitely support removing the version entirely.

**Comment by erasche** [comment]
Better, @nsoranzo?

File: tools/rgrnastar/.shed.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 categories:
 - Next Gen Mappers
-description: rna STAR 2.0.3e
+description: rna STAR 2.4.0d
```
Comment (nsoranzo) [inline tools/rgrnastar/.shed.yml]: Better remove the version from here too?

**Comment by nsoranzo** [comment]
The `name` and `remote_repository_url` value in `packages/package_rnastar_2_4_0d/.shed.yml` should also be updated to `package_rnastar_2_4_0d`.

**Comment by erasche** [comment]
Great catches, thanks. Forgot about that package .shed.yml

**Comment by nsoranzo** [comment]
:+1: 

Thanks for fixing!

**Pull request merged**